test/framework/src/main/java/org/elasticsearch/test/rest/spec/RestApiParser.java

```diff
@@ -21,119 +21,114 @@
 import org.elasticsearch.common.xcontent.XContentParser;
 
 import java.io.IOException;
+import java.nio.file.Path;
 
 /**
  * Parser for a REST api spec (single json file)
  */
 public class RestApiParser {
 
-    public RestApi parse(XContentParser parser) throws IOException {
+    public RestApi parse(Path path, XContentParser parser) throws IOException {
```

Comment:
sorry for the noise in this file, I simply moved parser.close out of it, which is now done in RestSpec#parseFrom using try with resource.